src/Sql/SqlBase.php

```diff
@@ -119,6 +119,15 @@ public static function create(array $options = []): ?SqlBase
     public static function getInstance($db_spec, $options): ?self
     {
         $driver = $db_spec['driver'];
+        // Handle custom database drivers.
```

Comment:
Expand comment to say that this is helpful when you use a custom Drupal DB driver and want to use a default Drush DB driver.